e2e/e2e_test.go

```diff
@@ -432,13 +468,12 @@ func Test_EVM_Hedera_Native_Token(t *testing.T) {
 	}
 
 	// Step 4: Validate that a scheduled token mint txn was submitted successfully
-	bridgeMintTransactionID, bridgeMintScheduleID := validateScheduledMintTx(setupEnv, setupEnv.BridgeAccount, setupEnv.TokenID.String(), mintTransfer, t)
+	bridgeMintTransactionID, bridgeMintScheduleID := verify.ScheduledMintTx(t, setupEnv.Clients.Hedera, setupEnv.Clients.MirrorNode, setupEnv.BridgeAccount, setupEnv.TokenID.String(), mintTransfer, now)
 
 	// Wait for validators to update DB state after Scheduled TX is mined
-	time.Sleep(30 * time.Second)
```

Comment:
I think we will need a more stable fix, as we already sync we could have a re-try in case of timeout.